### PR TITLE
Proposing Additional Resources

### PR DESCRIPTION
**Pull request by djscheuf**
Preparing to add several resource recommendations. Will attempt to list them here:

- Added Team Topologies to Genrealist Book Recommendations. Deals with managing team interactions, useful both for managing a single team, and for managing many team interactions.
- Added Simple Sabotage to Meetings Recommendation. Calls out Collaboration anti-patterns, and practical remediations for these. Builds on CIA's Sabotage field manual provided to French resistance during the Occupation of World War II.
  - Propose a YT video 'Designing and Running Effective Meetings', which builds on Simple Sabotage, providing practical proactive steps to avoid some of the anti-patterns. Full Disclosure, this Video is one of my presentations. I created it specifically for this kind of audience. However, I recognize the potentially dubious nature. I submit to the cheif maintainer for an honest evaluation. If it does not fit their vision, or add to this subject area, I will happily rescind the recommendation. Just let me know. 
- Added Liberating Structures to Meetings. These are a series of Meeting formats, designed to improve collaboration and engagement. Strongly recommend, based on personal practice and ability to radically change meeting dynamics!
- Added The Effective Executive, by Peter Drucker to Generalist Book Recomendations. Seminal work in Management, and specifically deals with managing Knowledge Workers (i.e. Engineers). Would also serve as a sound basis for future career growth.

Hoping for guidance on appropriate location to add these. Also proposing one change per Commit to make it easier to isolate changes.

### CODE REVIEW

**Comment by djscheuf** [comment]
@charlax As you consider these changes, please accept my heart-felt thanks for assembling the master list! I have benefitted from your recommendations on several occasions. I wish to contribute to this resource, for future leaders in our space!

File: README.md [hunk 1]
```diff
@@ -167,6 +167,9 @@ You can find a short summary in video [here](https://www.youtube.com/watch?v=psA
...
 - 📖 Work Rules!: Insights from Inside Google That Will Transform How You Live and Lead, Laszlo Bock. A pretty interesting description of Google's processes. A bit long at times.
 - 📖 [The Manager's Path](https://www.oreilly.com/library/view/the-managers-path/9781491973882/) , Camille Fournier. A very practical book with lots of down-to-earth advices.
+- 📖 [Team Topologies](https://itrevolution.com/product/team-topologies/), from ITRevolution Press. Discusses the intricacies of managing Engineering Departments, and especially patterns for improving team interactions.
+- 📖 [The Effective Executive](https://www.amazon.com/Effective-Executive-Definitive-Harperbusiness-Essentials/dp/0060833459) by Peter Drucker, Seminal Work in Management. Discusses challenges of management, especially managing Knowledge Workers. Proposes principles for effective decision-making and continued improvement of one's organization
+  - May also serve as grounding for growing beyond Engineering Management, as well as working with other departments.
```
Comment (charlax) [inline README.md:172]: Those ones make a ton of sense, thanks!

File: README.md [hunk 2]
```diff
@@ -991,6 +995,9 @@ Quote:
 - ["My new favorite team ritual: A weekly meeting called "Fight Club" where you meet with your leadership team with the intention of having a conflict.](https://twitter.com/lennysan/status/1754554023318106596)
 - [Meetings *are* the work.](https://medium.com/@ElizAyer/meetings-are-the-work-9e429dde6aa3)
 - [Embrace Silence](https://tech.forter.com/embrace-silence.html)
+- [Simple Sabotage](https://www.amazon.com/Simple-Sabotage-Detecting-Behaviors-Undermine/dp/0062371606) "A Modern Field Manual for Detecting and Rooting Out Everyday Behaviors That Undermine Your Workplace". Covers anti-patterns of collaborative behavior, and provides specific recommendations for fixing these when they occur in meetings.
```
Comment (charlax) [inline README.md:998]: This one is great, and a classic.

File: README.md [hunk 3]
```diff
@@ -991,6 +995,9 @@ Quote:
...
+  - 🎞 [Designing and Running Effective Meetings](https://www.youtube.com/watch?v=rN5UJ4J36YY): "Some simple practices to make all your meetings more effective, and not just the ones you lead!"
+- [Liberating Structures](https://www.liberatingstructures.com/ls/), for when you need people engaging with hard problems! "Liberating Structures introduce tiny shifts in the way we meet, plan, decide and relate to one another. They put the innovative power once reserved for experts only in hands of everyone."
```
Comment (charlax) [inline README.md:1000]: I don't think those ones add much, can you remove them? thanks!

**Review by charlax — COMMENTED** [review]
Thanks for your contribution! Left a bunch of comments

File: README.md (hunk 3 above)
Comment (djscheuf) [reply README.md]: @charlax Trimmed, though I do strongly advocate you look into Liberating Structures. Thanks for looking these over.

**Comment by djscheuf** [comment]
@charlax Comments addressed. Ready to merge at your call.

**Comment by charlax** [comment]
Merged, thanks!

**Pull request merged**